src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -121,6 +121,18 @@ describe('Repeater item component', () => {
     flushDropdownTimer();
   }));
 
+  it('hide the chevron and disable expand/collapse for items with no content', fakeAsync(() => {
```

Comment:
@blackbaud-GavinNicol This is looking great. Just one more request - could you please also write a test for showing the chevron once content items are dynamically added? Thanks!

Reply:
Yup, added a test for it. I'm wondering if maybe it makes sense to just combine the two tests into one since the one I just added basically includes what the first test was checking for as well.

Reply:
Thanks! I think two tests are OK. They are specifically testing for something different.